packages/connect/src/protocol-grpc-web/transport.ts

```diff
@@ -71,9 +71,11 @@ export function createTransport(opt: CommonTransportOptions): Transport {
         opt.jsonOptions,
         opt
       );
-      const ac = createLinkedAbortController(signal);
-      return await runUnary<I, O>(
```

Comment:
The new `run*` functions handle abort signals and timeouts. They also trigger the abort signal if the Transport raises an error. This saves quite a bit of plumbing in Transport implementations, and seems worth the change.